### DISCUSSION TITLE
GitHub Projects - Do we need this?

### DISCUSSION THREAD

**Discussion by andregasser** [andregasser/bigbone#60] [General]
I have played around with the "Projects" feature on this Github repo and created a "2.0.0 Release" project. Not sure if it is really worth to maintain a project, when we have our milestones. Milestones can track progress also. I'll probably just delete it again, as I don't see any benefits here... Opinions?

**Comment by bocops** [comment]
Yeah, I don't think we need projects to duplicate milestones. A dedicated project could perhaps be useful for something that is bigger than a single issue, but not yet the same as a release milestone - for example the various "Add missing fields to ..." issues (#14 - #24) could be listed in a project.

Not sure how necessary that is at the moment, though. :)

**Reply by andregasser** [reply]
I'll delete it.
